contracts/access/manager/IAccessManager.sol

```diff
@@ -29,7 +29,7 @@ interface IAccessManager {
     event OperationCanceled(bytes32 indexed operationId, uint32 indexed nonce);
 
     event GroupLabel(uint64 indexed groupId, string label);
-    event GroupGranted(uint64 indexed groupId, address indexed account, uint32 delay, uint48 since);
+    event GroupGranted(uint64 indexed groupId, address indexed account, uint32 delay, uint48 since, bool isUpdate);
```

Comment:
I had originally thought we would emit the opposite of this. Something like `bool newMember` that is true when the grant is an actual grant. What do you think?

Reply:
Its symetric, so either is good to me.

Reply:
changed